web-app-router/components/Input/index.tsx

```diff
@@ -87,7 +89,8 @@ export const Input = memo(function Input(props: InputInterface) {
         <div className="flex items-center">{addOnRight && addOnRight}</div>
 
         <legend className={labelClassNames}>
-          {label} {required && <span className="text-system-error-500">*</span>}
+          {labelNode ? labelNode : label}{" "}
```

Comment:
@andy-t-wang Maybe it's a good idea to update the label type to `ReactNode`? I think string matches the `ReactNode` type